Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -585,3 +602,48 @@ class SettingsTableViewController: UITableViewController {
         return nil
     }
 }
+
+private class SettingsTableSectionHeaderView: UIView {
+    lazy var titleLabel: UILabel = {
+        var headerLabel = UILabel()
+        var frame = headerLabel.frame
+        frame.origin.x = 15
+        frame.origin.y = 17
+        headerLabel.frame = frame
+        headerLabel.textColor = AppConstants.TableViewHeaderTextColor
+        headerLabel.font = AppConstants.DefaultStandardFontBold
+        return headerLabel
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.backgroundColor = AppConstants.TableViewHeaderBackgroundColor
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private override func drawRect(rect: CGRect) {
+        var topBorder = CALayer()
+        topBorder.frame = CGRectMake(0.0, 0.0, rect.size.width, 0.5)
+        topBorder.backgroundColor = AppConstants.SeparatorColor.CGColor
+        layer.addSublayer(topBorder)
+        var bottomBorder = CALayer()
+        bottomBorder.frame = CGRectMake(0.0, rect.size.height - 0.5 , rect.size.width, 0.5)
+        bottomBorder.backgroundColor = AppConstants.SeparatorColor.CGColor
+        layer.addSublayer(bottomBorder)
+    }
+
+    private override func layoutSubviews() {
+        if let label = titleLabel.text {
+            if titleLabel.superview == nil {
+                addSubview(titleLabel)
+            }
+            titleLabel.sizeToFit()
+        }
+        else {
+            titleLabel.removeFromSuperview()
```

Comment:
Hm, a few questions about this method:
 1. Why is `titleLabel` being created lazily and attached in `layoutSubviews()`, as opposed to simply being added as a subview in `init()`?
 1. Why are we removing it here?